### PR TITLE
Shaping the use-case around quality gates to emphasize release/deployment validation

### PR DESCRIPTION
**Pull request by johannes-b**
Signed-off-by: Johannes <johannes.braeuer@dynatrace.com>

### CODE REVIEW

File: content/docs/concepts/quality_gates/index.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ weight: 2
 keywords: [keptn, use-cases]
 ---
 
-> When developing a software product, sooner or later you need to update a service in a production environment. To conduct this in a controlled manner and without impacting end-user experience, the quality of the new service has to be ensured in an automated way. 
+> When developing a software product, sooner than later you need to deliver a new deployment of your service in a production environment. To conduct this in a controlled manner and without impacting end-user experience, the quality of the new service has to be ensured in an automated way.
```
Comment (christian-kreuzberger-dtx) [inline content/docs/concepts/quality_gates/index.md:8]: I would suggest to say "deliver a new version of your service". You usually don't deliver a deployment.

File: content/docs/concepts/quality_gates/index.md [hunk 2]
```diff
@@ -5,11 +5,11 @@ weight: 2
...
+> When developing a software product, sooner than later you need to deliver a new deployment of your service in a production environment. To conduct this in a controlled manner and without impacting end-user experience, the quality of the new service has to be ensured in an automated way. 
 
-:bulb: *A quality gate answers one question: Does my service meet all defined quality criteria?*
+:bulb: *A quality gate allows conducting a deployment/release validation by ensuring that defined quality criteria are met.*
 
-Keptn quality gates provide you a *declarative way* to define quality criteria of your service. Therefore, Keptn will collect, evaluate, and score those quality criteria to decide if a new version is allowed to be promoted to the next stage or if it has to be held back.
+Keptn quality gates provide you a *declarative way* to define the quality criteria of your service. Therefore, Keptn will collect, evaluate, and score those quality criteria to decide if a new deployment or release is allowed to be promoted to the next stage or if it has to be held back.
```
Comment (christian-kreuzberger-dtx) [inline content/docs/concepts/quality_gates/index.md:12]: Remove the in "define ~the~ quality criteria", criteria is plural in this case (or is it? :D)

> to decide if a new deployment or release is

I also think we should not refer to "deployment" here. I'd rather refer to "version" or "release".

File: content/docs/concepts/performance_validation/index.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ Keptn helps automating your tests by having Keptn triggering the test execution
 
 ## Test Automation Process
 
-The following image shows the general process of how Keptn is providing continuous performance verification. An underlying core concept for this process it the SLO/SLI validation as explained in [Quality Gates](../quality_gates/).
+The following image shows the general process of how Keptn is providing continuous performance verification. An underlying core concept for this process it the SLO/SLI-based deployment and release validation as explained in [Quality Gates](../quality_gates/).
```
Comment (christian-kreuzberger-dtx) [inline content/docs/concepts/performance_validation/index.md:14]: Same as mentioned below, I don't think you can validate a deployment. I'd rather validate a version or a release.

**Review by oleg-nenashev — COMMENTED** [review]
+1 to the feedback by @christian-kreuzberger-dtx except the "deployment validation". This is quite common term in SRE, and it's also used in application monitoring systems including but not limited to Dynatrace. IMHO it's fine to keep it https://www.google.com/search?q=%22deployment+validation%22

File: content/docs/concepts/quality_gates/index.md (hunk 1 above)
Comment (johannes-b) [reply content/docs/concepts/quality_gates/index.md:8]: changed it :)

File: content/docs/concepts/performance_validation/index.md (hunk 3 above)
Comment (johannes-b) [reply content/docs/concepts/performance_validation/index.md:14]: I agree with you that promoting/delivering a deployment does not make sense (see your comments above). However, I still see it valid to conduct a deployment validation. Kind of pre-step before releasing the new version. Hence, I would like to leave this as it is.

**Review by oleg-nenashev — APPROVED** [review]

**Pull request merged**